pkg/comp-functions/runtime/runtime.go

```diff
@@ -173,3 +174,16 @@ func parseCompFuncConfig(iof *Runtime) (*corev1.ConfigMap, error) {
 func (r *Runtime) GetRawFuncIO() *xfnv1alpha1.FunctionIO {
 	return &r.io
 }
+
+// GetBoolFromConfigMap is a small function to help us retrieve bool values from configMap
+func (r *Runtime) GetBoolFromConfigMap(key string) bool {
```

Comment:
Can we rename this function? `GetBoolFromConfigMap` sounds confusing as it doesn't indicate what config map is meant.

Maybe a name like `GetBootFromCompositionConfig` would be more helpful.

Reply:
applied it